app/src/main/java/org/sopt/carrot/data/Service/DummyService.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.carrot.data.Service
```

Comment:
p1: 패키지명 수정 부탁드립니다~!
```suggestion
package org.sopt.carrot.data.service
```

Reply:
헐 뭐야!!!! 그렇네?!?! 눈썰미 미쳤다;;;;;